Scribe/AboutTab/AboutTableData.swift

```diff
@@ -9,28 +9,28 @@ struct AboutTableData {
     ParentTableCellModel(
       headingTitle: "Community",
       section: [
-        Section(sectionTitle: "See the code on GitHub", imageString: "github", hasToggle: false, sectionState: .github),
-        Section(sectionTitle: "Chat with the team on Matrix", imageString: "matrix", hasToggle: false, sectionState: .matrix),
-        Section(sectionTitle: "Wikimedia and Scribe", imageString: "wikimedia", hasToggle: false, sectionState: .wikimedia),
-        Section(sectionTitle: "Share Scribe", imageString: "square.and.arrow.up", hasToggle: false, sectionState: .shareScribe),
+        Section(sectionTitle: "See the code on GitHub", imageString: "github", sectionState: .github),
+        Section(sectionTitle: "Chat with the team on Matrix", imageString: "matrix", sectionState: .matrix),
+        Section(sectionTitle: "Wikimedia and Scribe", imageString: "wikimedia", sectionState: .wikimedia),
+        Section(sectionTitle: "Share Scribe", imageString: "square.and.arrow.up", sectionState: .shareScribe),
       ],
       hasDynamicData: nil
     ),
     ParentTableCellModel(
       headingTitle: "Feedback and support",
       section: [
-        Section(sectionTitle: "Rate Scribe", imageString: "star", hasToggle: false, sectionState: .rateScribe),
-        Section(sectionTitle: "Report a bug", imageString: "ant", hasToggle: false, sectionState: .bugReport),
-        Section(sectionTitle: "Send us an email", imageString: "envelope", hasToggle: false, sectionState: .email),
-//        Section(sectionTitle: "Reset app hints", imageString: "lightbulb", hasToggle: false, sectionState: .appHints)
+        Section(sectionTitle: "Rate Scribe", imageString: "star", sectionState: .rateScribe),
+        Section(sectionTitle: "Report a bug", imageString: "ant", sectionState: .bugReport),
+        Section(sectionTitle: "Send us an email", imageString: "envelope", sectionState: .email),
+//        Section(sectionTitle: "Reset app hints", imageString: "lightbulb", sectionState: .appHints)
       ],
       hasDynamicData: nil
     ),
     ParentTableCellModel(
       headingTitle: "Legal",
       section: [
-        Section(sectionTitle: "Privacy policy", imageString: "lock.shield", hasToggle: false, sectionState: .privacyPolicy),
-        Section(sectionTitle: "Third-party licenses", imageString: "thirdPartyLicenses", hasToggle: false, sectionState: .licenses),
+        Section(sectionTitle: "Privacy policy", imageString: "lock.shield", hasNestedNavigation: true, sectionState: .privacyPolicy),
```

Comment:
Switching it to nested navigation as a check makes a lot more sense, @damien-rivet :)